### PR TITLE
JBPM-7664 Tasks: Task Delegation help message is incorrect

### PR DESCRIPTION
**Pull request by bxf12315**

### CODE REVIEW

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties [hunk 1]
```diff
@@ -94,8 +94,8 @@ ProcessSessionId=Process Session Id
 WorkItemId=Work Item Id
 TaskSuccessfullyForwarded=Task was successfully forwarded
 ReminderSentTo=Reminder was successfully sent to {0}
-PleaseEnterUserOrAGroupToDelegate=Please enter a user or a group to delegate the task
-PleaseEnterUserOrAGroupToForward=Please enter a user or a group to forward the task
+PleaseEnterUserOrAGroupToDelegate=Please enter a user to delegate the task
+PleaseEnterUserOrAGroupToForward=Please enter a user to forward the task
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties]: The forward message is correct, only delegate needs to be changed.

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties [hunk 2]
```diff
@@ -94,8 +94,8 @@ ProcessSessionId=Process Session Id
 WorkItemId=Work Item Id
 TaskSuccessfullyForwarded=Task was successfully forwarded
 ReminderSentTo=Reminder was successfully sent to {0}
-PleaseEnterUserOrAGroupToDelegate=Please enter a user or a group to delegate the task
-PleaseEnterUserOrAGroupToForward=Please enter a user or a group to forward the task
+PleaseEnterUserOrAGroupToDelegate=Please enter a user to delegate the task
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties]:
```suggestion
PleaseEnterUserToDelegate=Please enter a user to delegate the task
```

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties (hunk 1 above)
Comment (bxf12315) [reply jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties]: I find PleaseEnterUserOrAGroupToForward and PleaseEnterUserOrAGroupToDelegate has never been used

**Comment by cristianonicolai** [comment]
@bxf12315 please squash changes into a single commit

**Comment by bxf12315** [comment]
@cristianonicolai  Done

**Pull request merged**